src/main/resources/application.properties

```
@@ -140,3 +140,5 @@ cloudforms.manifest.0.vmworkloadinventory.filesContentPathContents=contents
 cloudforms.manifest.0.vmworkloadinventory.productPath=$.ManageIQ::Providers::Vmware::InfraManager.emstype_description
 cloudforms.manifest.0.vmworkloadinventory.versionPath=$.ManageIQ::Providers::Vmware::InfraManager.api_version
 cloudforms.manifest.0.vmworkloadinventory.hostNamePath=$.ManageIQ::Providers::Vmware::InfraManager.hosts[?(@.id == {host_id})].hostname
+
+rest.authorization.administration=${admin.users}
```

This line prevents this PR to start in CI if there's no `admin.users` env property defined and we have to change this approach.
We need to provide a default so that the application always starts with high security level so change this to become:
```suggestion
rest.authorization.administration=
```
and then we'll take care of providig the list of authorized users in OCP.

---

Thanks @mrizzi , it is done